web/app/styles/graph-view.css

```diff
@@ -166,7 +166,8 @@ svg .bucket.axis-label {
 .graph-view-legend .inside {
   direction: ltr;
   width: 100%;
-  color: #39bcf3; /* Brand color. */
+  color: white; /* Brand color. */
```

Comment:
Is white a brand color? I guess now it is! 😄 "White™ by LynxKite"

Reply:
Haha, true enough! Fixed.